internal/db/schema/migrations/postgres/10/03_vault_credential.up.sql

```diff
@@ -209,15 +209,15 @@ begin;
         references credential_vault_http_method_enm (name)
         on delete restrict
         on update cascade,
-    http_request_body text
+    http_request_body bytea
```

Comment:
nit: We do plan on doing templating on this data eventually in our business logic.  In that sense this may make sense as a string, but it's not a big deal to cast it to a string when we are ready to process it.